sphinx/directives/other.py

```diff
@@ -39,6 +39,7 @@ class TocTree(Directive):
     final_argument_whitespace = False
     option_spec = {
         'maxdepth': int,
+        'name': str,
```

Comment:
Indeed, indent width for option and contents should be same level, it's a definition of reStructuredText
http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#directives